kategory-test/src/main/kotlin/kategory/laws/FunctorFilterLaws.kt

```diff
@@ -0,0 +1,31 @@
+package kategory.laws
+
+import io.kotlintest.properties.Gen
+import io.kotlintest.properties.forAll
+import kategory.*
+
+object FunctorFilterLaws {
+
+    inline fun <reified F> laws(AP: Applicative<F> = applicative(), FFF: FunctorFilter<F> = functorFilter(), EQ: Eq<HK<F, Int>>): List<Law> =
```

Comment:
What do you mean ? FunctorFilterLaws depend on FunctorLaws which need an Applicative to run, that's why it's being passed here, but the FunctorFilterLaws are being tested using the FunctorFilter passed as a second argument.